shell/list/provisioning.cattle.io.cluster.vue

```diff
@@ -167,6 +167,10 @@ export default {
         expiring: counts.expiring ? this.t('cluster.cloudCredentials.banners.expiring', { count: counts.expiring }) : '',
         expired:  counts.expired ? this.t('cluster.cloudCredentials.banners.expired', { count: counts.expired }) : '',
       };
+    },
+
+    showRke1DeprecationWarning() {
```

Comment:
This would still be a scaling issue (though minimum). Consider the edge / virtual cluster cases where churn would make this re-execute a lot.

Can it be moved into data/fetch to reduce the times it runs? This would mean if an rke1 cluster was created outside of the current tab the warning wouldn't pop up, but we would gain more in performance